tests/unit/drop-zone.spec.js

```diff
@@ -1,13 +1,13 @@
 import flatMap from 'lodash/flatMap';
+import XHRUpload from '@uppy/xhr-upload';
 import DropZone from '@/components/DropZone.vue';
 import { localMount } from './support';
 import Client from '../../src/client';
-import XHRUpload from '@uppy/xhr-upload';
 
 function assertUppyFiles(w, expected) {
   expect(
     flatMap(
-      flatMap(w.emittedByOrder(), e => e.args),
+      flatMap(w.emittedByOrder().filter(e => e.name !== 'client-initialized'), e => e.args),
```

Comment:
It does no make sense to filter by `client-initialized`. This function is meant to assert any files emitted by the client. So, I'd do something like:
``` javascript
function assertUppyFiles(w, expected) {
  expect(
    flatMap(
      flatMap(w.emittedByOrder(), e => e.args),
      l => l,
    ).map(u => get(u, 'meta.name').filter(Boolean),
  ).toEqual(expected);
}
```